soc/BatteryPower.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import sys
+import os.path
+
+class BatteryPower:
+    def __init__(self): 
+        self.resistance = 0.055
+
+    #based on BATTERY_AGGREGATE_VC data which has voltage + current out of battery pack
+    #unsure how to include the BATTERY_VT, which includes voltage + temperature out of one module?
+    def calc_power(self, current, voltage):
+        power = voltage * current
+        return power
+
+class TestBatteryPower:   
+    #testing using sample data from can_messages.csv
+    def test(self):
```

Comment:
Tests should go into a tests folder within the soc folder. See other modules as an example.

I won't block it and let Emma take a look.

Reply:
fixed! thanks